packages/cloud2edge/requirements.yaml

```diff
@@ -12,8 +12,8 @@
 #
 dependencies:
   - name: hono
-    version: ^1.10.18
+    version: ~2.0.7
```

Comment:
The chart changes for Hono 2.0 included renaming some K8s resources. We have dropped the `-vertx` suffix there.
An HTTP protocol adapter pod name now contains just `adapter-http` instead of `adapter-http-vertx`.

That means that in all the cloud2edge package files, in fact also in the homepage `prereqs.md`, all occurrences of `adapter-[*]-vertx` need to be replaced with `adapter-[*]`.

An example here is the 
`getService adapter-amqp-vertx "amqp amqps" AMQP_ADAPTER` 
line in `setCloud2EdgeEnv.sh`.

Reply:
done :)